descriptor.md

```diff
@@ -51,8 +56,14 @@ The following fields contain the primary properties that constitute a Descriptor
   The decoded data MUST be identical to the referenced content and SHOULD be verified against the [`digest`](#digests) and `size` fields by content consumers.
   See [Embedded Content](#embedded-content) for when this is appropriate.
 
+- **`artifactType`** *string*
```

Comment:
The `artifactType` is defined twice. Should we merge both or drop one?

Reply:
Opened https://github.com/oci-playground/image-spec/pull/14